index.js

```diff
@@ -55,6 +56,11 @@
   }
   extend(Nothing, Maybe);
 
+  //  Nothing#bind :: (a -> m b) -> m b
+  Nothing.prototype.bind = function(f) {  // jshint ignore:line
+    return this;
+  };
```

Comment:
this is the same as `map`, so you could use a reference to the same function if you were so inclined. not a big deal.